src/dlload.c

```diff
@@ -60,18 +60,18 @@ extern char *julia_bindir;
 
 #define JL_RTLD(flags, FLAG) (flags & JL_RTLD_ ## FLAG ? RTLD_ ## FLAG : 0)
 
-static void JL_NORETURN jl_dlerror(const char *fmt, const char *sym)
+static const char * jl_dlerror(void)
 {
 #ifdef _OS_WINDOWS_
-    CHAR reason[256];
+    static JL_THREAD_LOCAL CHAR reason[256];
```

Comment:
No change needed but note that the "more complex" tls implementation we have should actually be simpler and faster.

Reply:
Simpler in the code generated that is.